internal/controlplane/handlers_githubwebhooks_test.go

```diff
@@ -246,27 +243,6 @@ func (s *UnitTestSuite) TestHandleWebHookRepository() {
 	projectID := uuid.New()
 	providerID := uuid.New()
 
-	// create a test oauth2 token
```

Comment:
No longer needed because the token querying is not triggered unless the provider is actually instantiated. The test case for the handler does not need the provider to be instantiated.